packages/sync/src/class-actions.php

```diff
@@ -128,7 +128,7 @@ public static function add_sender_shutdown() {
 		) ) {
 			self::initialize_sender();
 			add_action( 'shutdown', array( self::$sender, 'do_sync' ) );
-			add_action( 'shutdown', array( self::$sender, 'do_full_sync' ) );
+			add_action( 'shutdown', array( self::$sender, 'do_full_sync' ), 9999 );
```

Comment:
`PHP_INT_MAX`

Reply:
I am not sold, we leave little room for hacks

Reply:
Also I always recommend against `PHP_INT_MAX` because if you wanna hook something after it, you can't.

Reply:
> Also I always recommend against PHP_INT_MAX because if you wanna hook something after it, you can't.

That's what I try to meant with "we leave little room for hacks" :)

Reply:
💪